.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default": true,
+  "MD004": false,
+  "MD013": false,
+  "MD025": false,
```

Comment:
This one could be enabled. Only `CONTRIBUTING.md` uses this, and we can easily nest the inner headings.